src/commands/junit/api.ts

```diff
@@ -36,6 +37,7 @@ export const uploadJUnitXML = (request: (args: AxiosRequestConfig) => AxiosPromi
   const spanTags = {
     service: jUnitXML.service,
     ...jUnitXML.spanTags,
+    '_dd.cireport_version': '2',
```

Comment:
NIT: I suppose it is to differentiate zipped input form unzipped ones.
Don't we have the version of the CLI for this maybe?

Reply:
indeed, it is for that. 

We could send the cli version indeed. We hadn't thought about it 😅 . It'd be a nice improvement though. For now we'll leave it as is, as the backend deployments take quite a bit